conformance/utils/suite/suite.go

```diff
@@ -126,6 +125,7 @@ type ConformanceOptions struct {
 	MeshManifests        string
 	NamespaceLabels      map[string]string
 	NamespaceAnnotations map[string]string
+	ReportOutputPath     string
```

Comment:
This variable is neither assigned with the flag value nor used in the suite's proper field.

Reply:
It's populated from the flag here  - https://github.com/kubernetes-sigs/gateway-api/pull/2864/files/3ae59a7f8680f862a6e483789f8e78a2d0fd1f11#diff-eb3b4f37d6de9237b962307f425e6cf2ab994b5a107611d32c40fc5e27c7bdd4R87

And it's used in `conformance.go` where the report is written out optionally.

Reply:
Since the suite doesn't write the report to disk I didn't think it necessary to have a property in the suite `struct`

Reply:
👍